dsl_parser/constraints.py

```diff
@@ -21,7 +21,8 @@
 from dsl_parser.constants import (
     INPUTS,
     DEFAULT,
-    CONSTRAINTS as CONSTRAINT_CONST
+    CONSTRAINTS as CONSTRAINT_CONST,
+    TYPE
```

Comment:
A trailing comma here will mean the next time we add something to this it only changes one line.